src/dsp/pdr.c

```diff
@@ -682,17 +922,6 @@ void pldm_entity_association_tree_visit(pldm_entity_association_tree *tree,
 	entity_association_tree_visit(tree->root, *entities, &index);
 }
 
-static void entity_association_tree_destroy(pldm_entity_node *node)
```

Comment:
why is this removed

Reply:
it is not removed , juts placed in a differnt place in the same file

Reply:
yes, why is that change needed?

Reply:
Because in C i cannot call a static function from another function if it is not placed above the calling function(compiled before the compile function) . I added another function to this file that called the static api so I had to place it in a diff place. C++ does not have this requirement, correct me if I am wrong